docs/java/how-to-register-namespace-in-java.md

```diff
@@ -0,0 +1,37 @@
+---
+id: how-to-register-namespace-in-java
```

Comment:
@rachfop any chance we could get Namespace creation examples for the other languages?

Reply:
I'm adding the topic for Go.